_data/site_menu.yml

```diff
@@ -31,47 +31,58 @@ installation:
       path: "/docs/reference/latest/installing_on_scvmm/index.html"
       desc: "How to install and configure the ManageIQ Management Engine appliance on Microsoft System Center Virtual Machine Manager (SCVMM)"
 
-appliances:
-  title: Appliances
+upgrading:
+  title: Upgrading
   children:
-    - title: Backup and Restore
-      path: "/docs/reference/latest/appliances/backup_and_restore.html"
-      desc: "How to backup and restore appliances"
     - title: Upgrading
       path: "/docs/reference/latest/appliances/upgrading.html"
       desc: "How to upgrade appliances"
 
-configuration:
-  title: Configuration
-  children:
-    - title: Deployment Planning Guide
-      path: "/docs/reference/latest/deployment_planning_guide/index.html"
-      desc: "ManageIQ Deployment Planning Guide"
-    - title: General Configuration
-      path: "/docs/reference/latest/general_configuration/index.html"
-      desc: "ManageIQ General Configuration"
-    - title: High Availability Guide
-      path: "/docs/reference/latest/high_availability_guide/index.html"
-      desc: "ManageIQ High Availability Guide"
-    - title: Appliance Hardening Guide
-      path: "/docs/reference/latest/appliance_hardening_guide/index.html"
-      desc: "ManageIQ Appliance Hardening Guide"
-
 administration:
   title: Administration
+  children:
+    - title: Provisioning Virtual Machines and Hosts
+      path: "/docs/reference/latest/provisioning_virtual_machines_and_hosts/index.html"
+      desc: "Provisioning, workload management, and orchestration in ManageIQ"
+    - title: Service Provisioning Using an Orchestration Template
+      path: "/docs/reference/latest/service_provisioning_using_an_orchestration_template/index.html"
+      desc: "How to create orchestration template to deploy services"
+    - title: Creating a Service in for Virtual Machine Provisioning
+      path: "/docs/reference/latest/creating_a_service_for_vm_provisioning/index.html"
+      desc: "How to create a service for virtual machine provisioning"
+    - title: Assigning a Custom Analysis Profile to a Virtual Machine
+      path: "/docs/reference/latest/assigning_custom_analysis_profile_to_vm/index.html"
+      desc: "How to assign custom analysis profile to a virtual machine"
+    - title: Backup and Restore (appliances)
+      path: "/docs/reference/latest/appliances/backup_and_restore.html"
+      desc: "How to backup and restore appliances"
+
+visualizing:
+  title: Visualizing infrastructure
```

Comment:
Hm I'm not sure "visualizing" is how I'd describe this section.  It does include viewing inventory but also e.g. reconfiguring_a_vm_disk.md, refreshing_instances_and_images.md, removing_a_datastore.md, creating_a_vm_snapshot.md, etc...

If anything I'd call this "Inventory" or "Managing Inventory", @Fryguy wdyt?

Reply:
Oh actually that's what the section used to be called, https://github.com/ManageIQ/manageiq-documentation/pull/1806/files#diff-46686079121737af3636a6d1c95d842fc981118049ec6ece21ce578576eebd50L71

Reply:
Oh I see, I can make the top level heading `Managing inventory`.